rpm/kf5bluezqt-bluez5.spec

```diff
@@ -53,6 +54,13 @@ Requires:   %{name} = %{version}
 %description devel
 This package contains the development header files for kf5bluezqt
 
+%package doc
+Summary:    API documentation for %{name}
+BuildArch:  noarch
```

Comment:
Is no BuildArch here needed? This is the first package of this kind where I see it.

Reply:
It's pretty much in all of our documentation packages.